rero_ils/modules/items/api/issue.py

```diff
@@ -174,6 +174,14 @@ def _process_issue_claim(
         :return a count of modified issues.
         """
         days = cls._get_issue_status_age(issue)
+        print(
+            'yyyyyy',
+            issue.pid,
+            days,
+            claims_days,
+            days >= claims_days,
+            issue.get('issue')
+        )
```

Comment:
Seems to be debug ? If yes, it can be removed

Reply:
Ok, deleted.